src/components/ScrollTable/ScrollTable.tsx

```diff
@@ -0,0 +1,118 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { StandardProps, omitProps } from '../../util/component-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import Table from '../Table/Table';
+
+const cx = lucidClassNames.bind('&-ScrollTable');
+
+const { object, string, bool, node, number, oneOfType } = PropTypes;
+
+const defaultProps = {
+	hasWordWrap: false,
+	hasBorder: false,
+};
+
+export interface IScrollTableProps
+	extends StandardProps,
+		React.DetailedHTMLProps<
+			React.HTMLAttributes<HTMLDivElement>,
+			HTMLDivElement
+		> {
+	/** Set the width of the Table inside the scrollable container. */
+	tableWidth?: number | string;
+
+	/** Set the Table contents to not allow word wrapping. */
+	hasWordWrap: boolean;
+
+	/** render the table with borders on the outer edge. */
+	hasBorder: boolean;
+}
+
+export const ScrollTable = (props: IScrollTableProps): React.ReactElement => {
+	const {
+		children,
+		className,
+		style,
+		tableWidth,
+		hasWordWrap,
+		hasBorder,
+		...passThroughs
+	} = props;
+
+	return (
+		<div
+			className={cx(
+				'&',
+				{
+					'&-has-border': hasBorder,
+				},
+				className
+			)}
+			style={style}
+		>
+			<Table
+				{...omitProps(
+					passThroughs,
+					undefined,
+					_.keys(ScrollTable.propTypes),
+					false
+				)}
+				style={{
+					width: tableWidth,
```

Comment:
Will this not fail without a `defaultProp`?

Reply:
Looks like React can detect that `tableWidth` is undefined and will ignore this key/val pair. TIL!